src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible']}
+  }
+
+
+handleChange(event) {
+    this.setState({value: event.state.value});
+  }
+
+
+  handleClick(){
+    if(this.state.value.length > 0){
+    this.state({comments: _.concat(this.state.comments, this.state.value)})
```

Comment:
`this.state` isn't a function (your console will probably be erroring on this). You should be using the same function as line #12